src/rhsm/connection.py

```diff
@@ -752,7 +754,19 @@ def _create_connection(self, cert_file: str = None, key_file: str = None) -> htt
                 log.debug("Reusing connection: %s", self.__conn.sock)
                 return self.__conn
 
-        log.debug("Creating new connection")
+        # The original code is always six function calls deep:
+        # _create_connection()
+        #   _make_request()
+        #     _request()
+        #       _request_post()/_request_get()/...
+        #         API wrapper function
+        #           ...the code we are interested in...
+        #             two frozen functions
+        call_frames = inspect.stack()[6:-2]
+        log.debug(
+            "Creating new connection:\n"
+            + "\n".join(f"  {frame.filename}:{frame.function}:{frame.lineno}" for frame in call_frames)
+        )
```

Comment:
Isn't this duplicate of this: https://github.com/candlepin/subscription-manager/blob/main/src/rhsm/connection.py#L904
I'm sorry that it is not documented here: https://www.candlepinproject.org/docs/subscription-manager/debug_http_traffic.html, but it caused some issues with older versions of Python.